src/rust/engine/src/intrinsics.rs

```diff
@@ -118,6 +118,7 @@ fn multi_platform_process_request_to_process_result(
   args: Vec<Value>,
 ) -> BoxFuture<'static, NodeResult<Value>> {
   let core = context.core.clone();
+  let store = context.core.store();
```

Comment:
It shouldn't be necessary to clone this, because the `core` is also moving into this closure.